src/Controllers/MoviesController.cs

```diff
@@ -20,6 +23,39 @@ public IUnitOfWork UnitOfWork
 
         public IActionResult Index()
         {
+            var movies = _unitOfWork.MovieRepository.GetAll();
+
+            var moviesModels = new List<MovieViewModel>();
+
+            foreach (var m in movies)
+            {
+                var vm = new MovieViewModel { ID = m.ID, Title = m.Title, Genre = m.Genre, Price = m.Price, ReleaseDate = m.ReleaseDate };
```

Comment:
this works fine, but to do things like this, we use LINQ. for next PR, google it.

Reply:
![carbon](https://user-images.githubusercontent.com/19524560/45963759-4a0e9900-bffa-11e8-8b1c-bedc86f236d5.png)